main.go

```diff
@@ -87,6 +96,9 @@ func init() {
 	flag.IntVar(&config.RCache, "rcache", 0, "capacity of the response cache") // default to 0 for now
 	flag.IntVar(&config.RCacheTtl, "rcache-ttl", server.RCacheTtl, "TTL of the response cache")
 
+	// Ndots
+	flag.UintVar(&config.Ndots, "ndots", uintEnv("SKYDNS_NDOTS", server.Ndots), "How many labels a name should have before we allow forwarding")
```

Comment:
Just make it an `int`, you want need the extra uintEnv, in the `int`-type conversion. Do need a check that it is smaller than zero obviously